src/Core/Grid/Definition/Factory/CategoryGridDefinitionFactory.php

```diff
@@ -151,6 +151,14 @@ protected function getColumns()
                         'sortable' => false,
                     ])
             )
+            ->add(
+                (new DataColumn('products_count'))
```

Comment:
We need to make sure this extra column does not stretch too much the listing (for small screens for example). Adding columns can sometime make the grid too large, and we need to scroll horizontally 😅

Reply:
Yep ! I will add a note for the QA team. In all cases we can reduce the description column.
See screenshot https://github.com/PrestaShop/PrestaShop/issues/23514